melbourne-school-of-theology.csl

```diff
@@ -255,7 +294,7 @@
     <choose>
       <if type="bill book chapter graphic legal_case legislation motion_picture paper-conference report song" match="any">
         <choose>
-          <if is-numeric="edition">
+          <if is-numeric="edition note">
```

Comment:
That is an odd one.  I'm unsure how that snuck in there!  I must confess to using the Visual Editor almost exclusively, but do not remember slipping that in!  I shall remove it..